docs/reference/2-ods-api/how-to-guides/how-to-deploy-an-extension-plugin.md

```diff
@@ -38,47 +21,124 @@ option.
+packages:TPDMCorePostgreSqlPopulatedTemplate:PackageSource        https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json
+packages:TPDMCorePostgreSqlPopulatedTemplate:PackageVersion       7.2.194
+Plugin:Folder                                                     ../../Plugin
+Plugin:Scripts:0                                                  tpdm
+Plugin:Scripts:1                                                  sample
+Urls                                                              http://localhost:54746
+
+WARNING: The following settings are being overridden by the EdFi.Ods.WebApi project's user secrets:
+
+Plugin:Folder   ../../Plugin
+Plugin:Scripts:0 tpdm
+Plugin:Scripts:1 sample
+
+Invoke-NewDevelopmentAppSettings done in 2s.
+
+
+-----------------------
+    Install-Plugins
+-----------------------
+
+& D:\Ed-Fi-ODS-Implementation\Plugin\tpdm.ps1
+D:\Ed-Fi-ODS-Implementation\tools/nuget install EdFi.Suite3.Ods.Extensions.TPDM.Core.1.1.0.Standard -source https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json -outputDirectory D:\ed-fi\Ed-Fi-ODS-Implementation\Plugin -ExcludeVersion -version 7.2.159
+D:\Ed-Fi-ODS-Implementation\Plugin\EdFi.Suite3.Ods.Extensions.TPDM.Core.1.1.0.7.2.159
+& D:\Ed-Fi-ODS-Implementation\Plugin\sample.ps1
+D:\Ed-Fi-ODS-Implementation\tools/nuget install EdFi.Suite3.Ods.Extensions.Sample -source https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json -outputDirectory D:\ed-fi\Ed-Fi-ODS-Implementation\Plugin -ExcludeVersion -version 7.2.24
+D:\Ed-Fi-ODS-Implementation\Plugin\EdFi.Suite3.Ods.Extensions.Sample.7.2.24
+Install-Plugins done in 10s.
+```
 
 After the successful execution of initdev, you will find the sample extension in
 `<source directory>`\\Ed-Fi-ODS-Implementation\\Plugin folder.
 
-![Sample Extension Plugin](../../../../static/img/reference/ods-api/image2021-10-26_16-49-9.png)
+```none title="Ed-Fi-ODS-Implementation/Plugin Directory Listing"
+> EdFi.Suite3.Ods.Extensions.Sample.7.2.24
+> EdFi.Suite3.Ods.Extensions.TPDM.Core.1.1.0.7.2.159
+- homograph.ps1
+- profiles.sample.ps1
+- sample.ps1
+- tpdm.ps1
+```
 
 Run the "Ed-Fi-Ods.sln" solution from Visual Studio and verify that the sample
-data models are listed in your API landing page.
-
-![Sample Plugin API Landing Page](../../../../static/img/reference/ods-api/image2021-10-26_16-52-9.png)
+data models are listed in your API landing page ("Discovery API" endpoint).
+
+```json
+{
+    "version": "7.2",
+    "informationalVersion": "7.2",
+    "suite": "3",
+    "build": "7.2.1201.0",
+    "dataModels": [
+        {
+            "name": "Ed-Fi",
+            "version": "5.1.0",
+            "informationalVersion": "The Ed-Fi Data Model 5.1"
+        },
+        {
+            "name": "Sample",           <-- THIS IS NEW
```

Comment:
This was the only thing I could think of to highlight the new entry. In an image of course we could draw rectangle around it, or point a red arrow at it, etc. Here is how it renders. I wouldn't do this for JSON that is supposed to be copied and used somewhere - just for JSON that is for display only.

![image](https://github.com/user-attachments/assets/af47c320-046c-493e-a26d-902d576af53e)

Reply:
Nice workaround! Another thing I've seen is highlighting a line green that contains an added change, just like what we see when reviewing code - although not sure how to add that.